### DISCUSSION TITLE
Sondaggio: Di cosa parlerà il libro?

### DISCUSSION THREAD

**Discussion by Cadienvan** [Il-Libro-Open-Source/book#2] [Sondaggi]
Dopo una prima fase di brainstorming, i primi topic principali sono stati evidenziati e credo sia un buon momento per stabilire il primo punto cardine di questa avventura: Di cosa parlerà il libro?

Nelle possibilità di scelta ho racchiuso tutte le opzioni che sono arrivate dal #1.

GitHub mi chiede di stare sotto un certo numero di caratteri per il poll, quindi sotto troverete una piccola intro di ogni opzione.  

# Manuale sul mondo Open Source: Un compendio di buone pratiche condite da storie reali
Il tema che sembra vertere per la maggiore riguarda un "meta" libro Open Source sull'Open Source.  
Ogni capitolo si comporrebbe di un tema (Licensing, la prima contribution, maintainers, come contribuire efficacemente, ecc..) condito da una o più "storie reali" che possano raccontare delle sfaccettature riguardo a quel capitolo.  
A tal riguardo, l'unico nodo che sembra dividere la community riguarda la presenza o meno di un capitolo sulla storia dell'OSS.  

Guido suggerisce anche un capitolo sul tema "Open Source per il recruiting: Che impatto ha"

_Sostenuto da: [teschiopol](https://github.com/teschiopol), [Balastrong](https://github.com/Balastrong), [pietrolerro](https://github.com/pietrolerro), [crixo](https://github.com/crixo), [GuidoPenta](https://github.com/GuidoPenta)._

# Manuale del buon dev: Guide e case history su vari argomenti attorno al mondo dev
- Personal branding
- Overcommunication
- Lavoro in asincrono
- Importanza di [networking e reputation, non solo skill](https://twitter.com/ossia/status/1079942716480344065?s=19)
- Sviluppo OSS
- Intelligenza emotiva
- Smart working (Quello vero)
- Analisi
- Architettura
- Comunicazione
- Documentazione
Questi sono solo alcuni dei topic che difficilmente vengono trattati nei primi anni di carriera di un dev.  
Associando il taglio pratico, si potrebbe realizzare un compendio di buone pratiche e esempi reali o realistici di come applicarle, ma soprattutto descriverne poi i vantaggi o gli svantaggi

_Sostenuto da: [NoriSte](https://github.com/NoriSte), [dinb79](https://github.com/dinb79)._

# Un racconto sull'Intelligenza Artificiale.
Copio parola per parola il commento che ha sostenuto il tema perché ben scritto.  

Analizziamo le possibili implicazioni di sistemi AI sempre più avanzati ospitati su eventuali computer quantistici dalla potenza di calcolo praticamente illimitata. Me ne vengono in mente giusto un paio :

- Che venga data fin troppa fiducia a tali sistemi, come già accade, con conseguente svalutazione (ed in certi casi sostituzione) del lavoro umano;
- Come ulteriore conseguenza, una polarizzazione sempre più eccessiva delle ricchezze e del potere economico della popolazione globale.  

Volendo rimanere negativi, è molto plausibile che tutto questo possa contribuire in modo non indifferente al collasso della società umana per come la conosciamo.

_Sostenuto da: [Cb040719](https://github.com/Cb040719)._

# Formazione
Anche qui, copio pari-pari il commento perché rende bene l'idea del taglio che si vorrebbe dare.  
C'è chi si spertica e perde ore di lavoro per creare una repo open-source che raccolga link, video, risorse per avere un certo set di competenze, chi crea ore di video, chi scrive articoli, e così via... c'è troppa confusione.

Oggi, un ragazzo che aspira a diventare developer e sta studiando, ha tantissime strade davanti per poter approfondire le sue competenze, sarebbe bello avere una guida. Scritta appunto in modo open-source.

Una sorta di percorso, non quindi una repo verticale su come diventare frontend-dev oggi, o data scientist, ma qualcosa che da un incipit più generico ti indirizzi e ti porti a qualcosa di concreto, scritto da devs, di recente.

Che non siano articoli su come fare gli accordion con bootstrap o come selezionare un elemento del DOM con jQuery.
In rete o anche chatGPT stesso, suggerisce roba obsoleta a volte.

Oggi il mondo devs ha davvero tante sfaccettature, un libro che ti spiega cosa puoi diventare e cosa dovresti studiare per diventarlo, scritto da devs senza alcuno scopo di lucro e non da case editrici potrebbe essere una buona risorsa per uno studente alle prime armi e che si approccia a questo mondo.

Noi siamo senza filtri, non di parte, siamo gli unici che possiamo parlare ad un altro dev e spiegargli cosa lo aspetta. Non può farlo una casa editrice o un guru della comunicazione senza avere un secondo fine.

_Sostenuto da: [buggyzap](https://github.com/buggyzap)_

P.S. Non ho fissato un tempo per le risposte, direi che il sondaggio resterà aperto 1-2 settimane per permettere a chiunque abbia volontà di partecipare alla discussione. Proprio per questo ho lasciato anche un ultimo punto aperto di discussione.  

P.P.S. Ho volutamente escluso me stesso dal sostegno dei temi, non vorrei che essendo la persona che ha fatto partire la discussione si pensi erroneamente di voler "soddisfare" qualche mio capriccio. Darò comunque opinioni e voterò come qualsiasi altro contributor.

**Comment by GuidoPenta** [comment]
Voterei una delle prime due, ma se devo proprio scegliere punterei alla seconda

**Reply by serenasensini** [reply]
Anche perché tu potresti essere un grande contributor... Già mi vedo uno dei capitoli: "non è la RAL", di Guido Penta

**Comment by gguizza** [comment]
Ho votato la seconda perche' sulla prima conosco gia' delle risorse piuttosto buone e non credo ci sia troppo da aggiungere al tema.

**Reply by Saleiux** [reply]
Saresti cosí gentile da condividerci le risorse buone che già conosci?
